idaes/apps/grid_integration/tests/test_coordinator.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+import pyomo.environ as pyo
+from idaes.apps.grid_integration.bidder import Bidder
+from idaes.apps.grid_integration.tracker import Tracker
+from idaes.apps.grid_integration.coordinator import DoubleLoopCoordinator
+from test_tracker import TestingModel
+from test_bidder import TestingForecaster
+
+tracking_horizon = 4
+bidding_horizon = 48
+n_scenario = 3
+n_tracking_hour = 1
+
+
+@pytest.fixture
+def coordinator_object():
+
+    # create solver
+    solver = pyo.SolverFactory("cbc")
+
+    ## create trackers
+    # make a tracker
+    tracking_model_object = TestingModel(horizon=tracking_horizon)
+    thermal_tracker = Tracker(
+        tracking_model_object=tracking_model_object,
+        n_tracking_hour=n_tracking_hour,
+        solver=solver,
+    )
+
+    # make a projection tracker
+    projection_tracking_model_object = TestingModel(horizon=tracking_horizon)
+    thermal_projection_tracker = Tracker(
+        tracking_model_object=projection_tracking_model_object,
+        n_tracking_hour=n_tracking_hour,
+        solver=solver,
+    )
+
+    ## create a bidder
+    forecaster = TestingForecaster(horizon=bidding_horizon, n_sample=n_scenario)
+    bidding_model_object = TestingModel(horizon=bidding_horizon)
+    thermal_bidder = Bidder(
+        bidding_model_object=bidding_model_object,
+        n_scenario=n_scenario,
+        solver=solver,
+        forecaster=forecaster,
+    )
+
+    ## create coordinator
+    coordinator_object = DoubleLoopCoordinator(
+        bidder=thermal_bidder,
+        tracker=thermal_tracker,
+        projection_tracker=thermal_projection_tracker,
+    )
+
+    return coordinator_object
+
+
+@pytest.mark.unit
+def test_assemble_project_tracking_signal(coordinator_object):
+
+    gen_name = coordinator_object.bidder.generator
```

Comment:
Overall need to explain the `current_ruc_dispatch` signal and expected signals are all a constant price for this unit test